### PR TITLE
Deep comment link to select and mark comment in content

### PR DESCRIPTION
**Pull request by yurabakhtin**
**What kind of change does this PR introduce?** (check at least one)

- [x] Feature

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [ ] All tests are passing
- [x] Changelog was modified

**Other information:**
Issue: https://github.com/humhub/humhub/issues/4399

### CODE REVIEW

File: protected/humhub/modules/comment/models/Comment.php [hunk 1]
```diff
@@ -220,7 +221,11 @@ public static function GetCommentsLimited($model, $id, $limit = null)
 
             $query = Comment::find();
             $query->offset($commentCount - $limit);
-            $query->orderBy('created_at ASC');
+            if (Comment::findOne(['id' => $currentCommentId])) {
+                $query->orderBy('`comment`.`id` <= ' . ($currentCommentId + $limit - 1));
```
Comment (luke-) [inline protected/humhub/modules/comment/models/Comment.php:225]: Can you please additionally enfore integers? Maybe intval($currentCommentId) && intval($limit). Just to be paranoid ;-)

**Review by luke- — CHANGES_REQUESTED** [review]

File: protected/humhub/modules/comment/models/Comment.php (hunk 1 above)
Comment (yurabakhtin) [reply protected/humhub/modules/comment/models/Comment.php]: Done here https://github.com/humhub/humhub/pull/5168/commits/fec4ce7beb1f04b67002ab501037f8e3c5ff260f#diff-4fcd54238adfe8774ee607eeadd9ff1c821166188d5f9dabd37cac4e2729a218R225.

File: protected/humhub/modules/stream/widgets/StreamViewer.php [hunk 2]
```diff
@@ -131,6 +131,10 @@ public function getData()
             $result['stream-contentid'] = Yii::$app->request->getQueryParam('contentId');
         }
 
+        if (!empty(Yii::$app->request->getQueryParam('commentId'))) {
+            $result['stream-commentid'] = Yii::$app->request->getQueryParam('commentId');
```
Comment (luke-) [inline protected/humhub/modules/stream/widgets/StreamViewer.php:135]: We should enfore `int` here.

File: protected/humhub/modules/comment/activities/NewComment.php [hunk 3]
```diff
@@ -46,4 +52,16 @@ public function getDescription()
         return Yii::t('CommentModule.base', 'Whenever a new comment was written.');
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getUrl()
+    {
+        $url = parent::getUrl();
```
Comment (luke-) [inline protected/humhub/modules/comment/activities/NewComment.php:60]: Wouldn't it be easier to just refer to the Perma Link Controller here?

**Review by luke- — CHANGES_REQUESTED** [review]

File: protected/humhub/modules/stream/widgets/StreamViewer.php (hunk 2 above)
Comment (yurabakhtin) [reply protected/humhub/modules/stream/widgets/StreamViewer.php]: Done here https://github.com/humhub/humhub/pull/5168/commits/2f04aaf00a29d305222ffb5d7b14cc3cc8a52628#diff-d6e721d6bf4bd24f63b3fbe8e055241ad7bd92ce32ebaceb0c92cf94b2e09c98R135.

File: protected/humhub/modules/comment/activities/NewComment.php (hunk 3 above)
Comment (yurabakhtin) [reply protected/humhub/modules/comment/activities/NewComment.php]: Done here https://github.com/humhub/humhub/pull/5168/commits/2f04aaf00a29d305222ffb5d7b14cc3cc8a52628#diff-e6cc3c96dbcaa1978cf2e96970550e14f594d514429b5e74c25022617824fe28R61.

**Review by luke- — APPROVED** [review]

**Pull request merged**